lib/scripts.js

```diff
@@ -8,7 +8,16 @@ function startNewGame() {
 startNewGame()
 
 $('.catergory-text').text(`Category: ${game.getCategory()}`);
-$('.round-text').text(`Round: ${game.currentRound}`)
+$('.round-text').text(`Round: ${game.currentRound}`);
+
+function spin() {
+  $('.spinner').addClass('spin');
+  setTimeout(() => {
+    $('.spinner').removeClass('spin');
+  }, 2002)
+}
+
+$('.spin-btn').on('click', spin);
 
```

Comment:
super stoked you found a way to make it spin, and make the dom changes after the spin animation!!!!